Fix up README.md

cdrage opened the pull request:
A few grammar mistakes + cleans up part of the README where we reiterate
some information.

tnozicka commented on README.md:
```diff
@@ -2,34 +2,36 @@
-It is a curated higher level abstraction for specific Kubernetes resources.
-Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes.
-The developer is generally concerned with the application that is being developed.
+The goal of OpenCompose is to make it easier for developers to transition to Kubernetes.
```
To me *transition* seems like we are trying to attract developers already using other tools which in general ain't the case. Could you figure out a better word?

pradeepto replied:
Definitely want "on-board" or anything that means "on-board". OpenCompose doesn't assume that people are using anything else before. It is supposed to be "kubernetes first".

cdrage replied:
On-board would be a good fit. I'll make the change

tnozicka commented on README.md:
```diff
@@ -2,34 +2,36 @@
-Very simple idea, isn't it? Developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications on Kubernetes.
-The developer is generally concerned with the application that is being developed.
+The goal of OpenCompose is to make it easier for developers to transition to Kubernetes.
+OpenCompose is a curator of higher abstraction levels for specific Kubernetes resources.
```
Would it make sense to say:
*OpenCompose is a declarative higher level abstraction for specific Kubernetes resources.*
?

pradeepto replied:
Agreed with @tnozicka 's comment here.

cdrage replied:
Yup, that's better :)

tnozicka commented on README.md:
```diff
@@ -2,34 +2,36 @@
-Please send PRs to improve this project, file issues and RFEs against this repository.
+Very simple idea, isn't it? 
+A developer shouldn't have to learn various Kubernetes concepts just to test and deploy their applications.
+Focusing on the application that is being developed.
```
*Focusing* - is this a typo? To me that sentence sounds strange, but my English shouldn't be taken as a reference :D.

cdrage replied:
Ah yeah, *focus*

tnozicka commented on README.md:
```diff
@@ -42,13 +44,14 @@ services:
-To deploy your application to Kubernetes cluster just run `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
+This will create two Kubernetes files in current directory - `deployment-helloworld.yaml` and `service-helloworld.yaml`.
+
+To deploy your application to Kubernetes run - `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
```
`kubectl create -f={service,deployment}-helloworld.yaml`?

kadel replied:
I don't know about that. I would actually prefer how it was before. It is just more readable even for people that don't know anything about bash

tnozicka replied:
Ok, let's leave it as it was. I have overlooked and thought there this line was changed and that there two `kubectl create` calls before - hence my comment about more simplification.

What I would like here is for it to change from inline to:
```bash
kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml
```

It makes the line easier to copy-paste and IMO looks nicer

cdrage replied:
Alright

tnozicka commented:
@cdrage Thanks for making the README better!

cdrage committed:
Fix up README.md

A few grammar mistakes + cleans up part of the README where we reiterate
some information.

cdrage commented:
@tnozicka @pradeepto I pushed/updated the PR with the suggested changes :)

tnozicka reviewed (approved):
lgmt
